### DISCUSSION TITLE
Grid Layout for Budgets

### DISCUSSION THREAD

**Discussion by cdrum** [firefly-iii/firefly-iii#4957] [General]
Hi, I'm wondering if there are any plans for a grid layout (like spreadsheet, columns being months) for the budgets? The way I plan is I like to see the current and perhaps next year, see what I budgeted and spent for last year, and update next year's budget numbers on a monthly basis. Right now by seeing only 1 month at a time it's very hard to see the "whole picture".

Great product though!
Thanks!!

**Comment by JC5** [comment]
Yes, it's on my list for the new layout, it's the only thing I routinely use Google Sheets for. In the mean time, use the yearly default report to get an overview.

**Reply by cdrum** [reply]
that's great thanks. Yes, I hacked a google spreadsheet script using the firefly api to get me this. Obviously native is better, but this will suffice until the new layout is out. thanks for the great work!
